.code-samples.meilisearch.yaml

```
@@ -605,6 +605,15 @@ filtering_guide_3: |-
     .execute()
     .await
     .unwrap();
+filtering_guide_nested_1: |-
+  let results: SearchResults<MovieRatings> = client
+    .index("movie_rating")
+    .search()
+    .with_query("thriller")
+    .with_filter(r#"rating.users >= 90"#)
```

```suggestion
    .with_filter("rating.users >= 90")
```
I'm maybe wrong, but I think the # works for an interpreted strings like if you have " in it

---

You are right! These should be changed in all code-samples using them :( I'll make an issue about it